src/components/structures/UserSettings.js

```diff
@@ -613,8 +613,7 @@ module.exports = React.createClass({
 
     onLanguageChange: function(newLang) {
         if(this.state.language !== newLang) {
-            // We intentionally promote this to the account level at this point
-            SettingsStore.setValue("language", null, SettingLevel.ACCOUNT, newLang);
+            SettingsStore.setValue("language", null, SettingLevel.DEVICE, newLang);
```

Comment:
for the inevitable question: I have no idea why this got promoted to an account level setting. It's always been a local setting.

Reply:
hah, i love the idea of changing language on one device magically synchronising over all your accounts :D

Reply:
'love' in a "aaaaaargh no" kinda way